sql/core/src/test/resources/sql-tests/analyzer-results/double-quoted-identifiers.sql.out

```diff
@@ -111,6 +111,201 @@ org.apache.spark.sql.catalyst.parser.ParseException
 }
 
 
+-- !query
+select 1 from "not_""exists"
```

Comment:
do we have similar behavior for the backtick quoting?

Reply:
Yes, Spark already supports using `` inside a back-quoted identifier. Here is the grammar in SqlBaseLexer.g4 (BACKQUOTED_IDENTIFIER : '`' ( ~'`' | '``' )* '`' ;):
![image](https://github.com/user-attachments/assets/d5d87d37-832e-4d8c-b3fc-5c1cd14d84ad)